griptape/artifacts/audio_artifact.py

```diff
@@ -1,12 +1,26 @@
 from __future__ import annotations
 
-from attrs import define
+from attrs import define, field
 
-from griptape.artifacts import MediaArtifact
+from griptape.artifacts import BlobArtifact
 
 
 @define
-class AudioArtifact(MediaArtifact):
-    """AudioArtifact is a type of MediaArtifact representing audio."""
+class AudioArtifact(BlobArtifact):
+    """Stores audio data.
 
-    media_type: str = "audio"
+    Attributes:
+        format: The audio format, e.g. "wav" or "mp3".
+    """
+
+    format: str = field(kw_only=True, metadata={"serializable": True})
+
+    @property
+    def mime_type(self) -> str:
+        return f"audio/{self.format}"
+
+    def to_bytes(self) -> bytes:
+        return self.value
+
+    def to_text(self) -> str:
+        return f"Audio, format: {self.format}, size: {len(self.value)} bytes"
```

Comment:
> Just updated the docstring and added a missing implementation. This field already existed in a number of Artifacts, I just bumped to the top since we now have a to_bytes.

how does the top level encoding work here?

Reply:
Yeah it's not used at all. I really don't like using `init=False` to remove fields in subclasses. What about updating the docstring to explain this behavior?

Or do you think encoding doesn't belong at the top?

Reply:
i would say we could just pass encoding/encoding_error_handler directly to the to_bytes/to_text methods, and have them default to the same thing as whatever encode/decode defaults to.

Reply:
I'm now realizing Loaders also take an encoding and set it on the Artifact.

```
TextLoader(encoding="ascii").load("my_file.txt").to_bytes() # should use the Loader-provided `ascii` encoding
```

Maybe these fields should just stay on `TextArtifact` and `BlobArtifact`, but then we'd have to make `to_bytes` abstract which feels like a barrier for creating new Artifact types.